Sources/RuleBuilder/Dictionary.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Dictionary.swift
+//
+//
+//  Created by Yume on 2022/8/5.
+//
+
+import Foundation
+
+public struct Dictionary: Text, CodeText {
+    // MARK: Lifecycle
+
+    public init(_ dictionary: [String: String]) {
```

Comment:
Maybe `public init(fromDictionary dictionary: [String: String])` for clarity?

Reply:
Current `init` definition is fine, maybe we need more various `init`.